FAILURE_CONDITION.md

```
@@ -85,6 +87,55 @@ ERROR: Task 1 cannot use 'success' and 'failure' together.
+
+**✅ VALID - Use aggregate conditions for blocks:**
+```bash
+task=1
+type=parallel
+tasks=10,11
+max_parallel=2
+next=2 if min_success=2, 99 if any_success, 100
+```
+
+**Error messages:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)